firebase-appdistribution/test-app/src/main/res/layout/activity_main.xml

```diff
@@ -92,7 +92,6 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="Sample App!"
-        android:textColor="@color/design_default_color_on_secondary"
```

Comment:
I removed these colors since they were making the text not show up very well in night mode.